README.md

```diff
@@ -1 +1,267 @@
+- [Testing](#testing)
+- [Deployment](#deploying-the-scenario)
+- [Observability](#observability)
+
+## <a name="what-you-build"></a>  What you'll build
+
+You’ll build a web service with load balancing and failover mechanisms. To understand this better, you'll be mapping this with a real-world scenario of a book finding service. The book finding service will call one from the three identical bookstore backends to retrieve the book details. With this guide you'll be able to understand how failover and load balancing mechanisms helps to balance the load among all the available remote servers.
+
+![Load Balancer](images/load_balancer_image1.png)
+
+**Request book details from book search service**: To search a new book you can use the HTTP GET request that contains the book name as a path parameter.
+
+## <a name="pre-req"></a> Prerequisites
+ 
+- JDK 1.8 or later
+- [Ballerina Distribution](https://github.com/ballerina-lang/ballerina/blob/master/docs/quick-tour.md)
+- A Text Editor or an IDE 
+
+### Optional requirements
+- Ballerina IDE plugins ([IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina), [VSCode](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina), [Atom](https://atom.io/packages/language-ballerina))
+- [Docker](https://docs.docker.com/engine/installation/)
+
+## <a name="developing-service"></a> Developing the RESTFul service with circuit breaker
+
+### Before you begin
+
+#### Understand the package structure
+The project structure for this guide should be as the following.
+
+```
+├── booksearchservice
+│   └── book_search_service.bal
+└── bookstorebacked
+    └── book_store_service.bal
+```
+
+The `booksearchservice` is the service that handles the client orders to find books from bookstores. Book search service call bookstore backends to retrieve book details. You can see that the loadbalancing and failover mechanisms are applied when the book search service calls three identical backend servers.
+
+The `bookstorebacked` is an independent web service that accepts orders via HTTP POST method from `booksearchservice` and sends the details of the book back to the `booksearchservice`.
+
+### Implementation of the Ballerina services
+
+#### book_search_service.bal
+The `ballerina.net.http.resiliency` package contains the load balancer implementation. After importing that package you can create an endpoint with a load balancer. The `endpoint` keyword in Ballerina refers to a connection with a remote service. Here you'll have three identical remote services to load balance across. First, create a LoadBalancer end point by ` create resiliency:LoadBalancer` statement. Then you need to create an array of HTTP Clients that you needs to be Loadbalanced across. Finally, pass the `resiliency:roundRobin` argument to the `create loadbalancer` constructor. Now whenever you call the `bookStoreEndPoints` remote HTTP endpoint, it goes through the failover and load balancer. 
+
```

Comment:
Explain more about failover aspect. Also, this example doesn't really use the failover capability (https://ballerinalang.org/docs/by-example/http-failover). Please revisit this.